theme/legacy.go

```diff
@@ -0,0 +1,82 @@
+package theme
+
+import (
+	"image/color"
+
+	"fyne.io/fyne"
+)
+
+// FromLegacy returns a 2.0 Theme created from the given LegacyTheme data.
+//
+// Since: 2.0.0
+func FromLegacy(t fyne.LegacyTheme) fyne.Theme {
```

Comment:
Should this have a deprecation path?

Reply:
I don't know what you mean - this is the deprecation path.
If we mark this as deprecated in 2.0 then people won't want to use it.
The only normal alternative seemed like a TODO which didn't read well.
I could add a ticket to the 3.0 milestone perhaps? or to 2.0 with a "Not until later" mark?

Reply:
Right right, my comment wasn't clear (and also in the wrong place). `fyne.LegacyTheme` is the deprecation path for the old `fyne.Theme`, but will `fyne.LegacyTheme` eventually be deprecated and removed too? If so a ticket on the 3.0 milestone to do this would fine, but we should also warn devs in the usual way

Reply:
I'll open the ticket - but adding deprecation is something to do later. We had a previous discussion about how we want to avoid releasing deprecated APIs.

Reply:
I added a note, I think this is the best we can do for 2.0